src/heap_utils.cc

```diff
@@ -231,19 +232,103 @@ class BufferOutputStream : public v8::OutputStream {
   std::unique_ptr<JSString> buffer_;
 };
 
+namespace {
+static const int kMaxPath = 4096;
```

Comment:
I’d prefer to inline this into the definition of `filename`, i.e. `char filename[4096]`? This way it looks a bit like it provides the actual limit.

Reply:
done!